book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -8,24 +8,324 @@ This can involve hosting different events that are attended by people with lived
+It is important to consider how you will handle potential disclosure and what you will do to uphold the personal safety of an individual who discloses personal information unknowingly in a public forum at your event. 
+
+By creating a safeguarding policy that covers the reporting and escalation pathways and making this accessible to staff and attendees you will be better prepared to manage any events of disclosure and uphold the necessary safeguarding for your attendees. 
+
+Please also remember that safeguarding applies to both attendees and staff and it is important to consider the physical and psychological safety of staff at your event, as well as your attendees.
+
+(cl-inclusive-events-planning-before-event)=
+## Before the Event (a few weeks before)
+
+(cl-inclusive-events-planning-speaker-preparation)=
+### Speakers and Moderators 
+Consider how you will support your speakers, moderators, and attendees being welcomed and comfortable with the technology your event will use. 
+Many of the technologies used every day by some people have never been used by others. 
+
+We suggest preparing introductions or tutorials for the technologies used by your event. 
+You can also consider hosting a "practice" session with your speakers and moderators before the event to ensure their familiarity and comfort. 
+
+(cl-inclusive-events-planning-during-event)=
+## At the Event
+
+(cl-inclusive-events-planning-during-event-speakers)=
+### Speakers and Moderators
+For speakers and moderators, you should consider how you will support them before their sessions and if they would be best supported by having an opportunity to prepare in a "Green Room" or quiet room before the session. 
+This is especially important for people with lived experience who may prefer to take medication in private, or may be newer to public speaking than their research colleagues.
+
+Consider how you will set them up to hold a successful session whether that be hosting a practice session for them before attendees arrive, identifying some questions or topics they may want to avoid and helping to facilitate discussion, or by asking how they wish to be introduced to the attendees. 
+
+If your event overlaps with a meal time, consider if their ability to eat or take a break will be impacted. If your event is in person, try to communicate with catering and ensure that there is a meal or alternative food is sourced and held aside for them. 
+
+(cl-inclusive-events-planning-during-event-tech)=
+### Technology 
+Depending on your event and platform, it may be helpful to organize "backup" technology on the day of the event. For example having spare batteries and spare microphones, arranging an alternate host for Zoom, and providing PDFs instead of relying solely on online technology to communicate important information. 
```

Comment:
```suggestion
Depending on your event and platform, it may be helpful to organize "backup" technology on the day of the event. 
For example, having spare batteries and spare microphones, arranging an alternate host for Zoom, and providing PDFs instead of relying solely on online technology to communicate important information. 
```